### PR TITLE
Arriba 1.0.1

### PR DESCRIPTION
**Pull request by suhrig**
Dear bioconda team,

Please review my pull request to add Arriba (https://github.com/suhrig/arriba) to your repository.

Thank you,
Sebastian

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by suhrig** [comment]
@bioconda/core This is my first recipe and a new tool. Now that the checks have passed, can you kindly review the recipe? Thanks.

File: recipes/arriba/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# compile HTSlib and Arriba
+make -C htslib-1.8 CPPFLAGS="-DHAVE_LIBDEFLATE -I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" libhts.a
+make CXXFLAGS="-pthread -std=c++0x -O2 -Isource -Ihtslib-1.8/htslib -I$PREFIX/include" LDFLAGS="-L$PREFIX/lib -ldeflate -lz -llzma -lbz2"
```
Comment (dpryan79) [inline recipes/arriba/build.sh]: Is it possible to use htslib from bioconda rather than compiling it statically for this package?

Comment (suhrig) [reply recipes/arriba/build.sh]: I modified the recipe. HTSlib is still linked statically, but the bioconda version is used.

The build process failed with what looks like a transient error to me:

```
conda.CondaMultiError: HTTP 500 INTERNAL SERVER ERROR for url <https://conda.anaconda.org/bioconda/linux-64/htslib-1.8-h57f8fdf_2.tar.bz2>
```

I had the same issue in my local build environment and simply restarting solved the issue. Maybe the Anaconda servers are having some issues at the moment. Is there a way to trigger a new CircleCI check without a new commit?

Comment (dpryan79) [reply recipes/arriba/build.sh]: I triggered a rebuild. The trick is to login to circleci with your github credentials, then the option appears.

Comment (dpryan79) [reply recipes/arriba/build.sh]: Does it have to be statically linked? It's not really a problem, but it prevents updating with bug fixes by just updating htslib.

Comment (dpryan79) [reply recipes/arriba/build.sh]: I expect this is related to bioconda getting shifted to the anaconda CDN. I'll hit rerun until it either passes CI testing or an actually relevant error is produced.

Comment (suhrig) [reply recipes/arriba/build.sh]: > The trick is to login to circleci with your github credentials

Okay. I was looking for this button all over the place.

> Does it have to be statically linked?

Conceptually, no. But the build process of Arriba currently does not support dynamic linking. I will add this capability in the next release of the software.

> I'll hit rerun until it either passes

Thanks, but don't bother. I can do this myself.

**Comment by dpryan79** [comment]
Thanks @suhrig, nice job!

**Pull request merged**